### PR TITLE
Footer updates

### PR DESCRIPTION
**Pull request by shivam0110**
Made footer responsive/Mobile friendly; and also restructured files (images)

### CODE REVIEW

File: src/Component/Footer/Footer.jsx [hunk 1]
```diff
@@ -1,44 +1,81 @@
...
-          <a href="" className="link">Events</a>
-          <a href="" className="link">Members</a>
-          <a href="" className="link">Institute Site</a>
-        </div>
+          <div className="link-drop">
+            <div className="linkna">
+              LINKS
+              <hr className="underline"></hr>
+            </div>
+            <div class="link-toogle" onClick = {() => link_toogle()} >
```
Comment (pAditya198) [inline src/Component/Footer/Footer.jsx:58]: a piece of advise there is also something called bind function which can be an alternative here

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind

File: src/Component/Footer/Footer.jsx [hunk 2]
```diff
@@ -1,44 +1,81 @@
...
+              <i class="fa fa-angle-down fa-2x" aria-hidden="true" color="white"></i>
+            </div>
+          </div>
+          <div className="links" id="cyka">
+            <a href="" className="link"  style={{display: 'block'}}>Home</a>
```
Comment (pAditya198) [inline src/Component/Footer/Footer.jsx:63]: was `style={{display:'block'}}` necessary?

File: src/Component/Footer/Footer.jsx [hunk 3]
```diff
@@ -1,44 +1,81 @@
...
         <div className="footerlin">
         <div className="conna">
           CONTACT
           <hr className="underline"></hr>
         </div>
         <div className="contact">
-          <p className="clink">Email:</p>
-          <p className="clink">Phone:</p>
-          <p className="clink">Address:</p>
+          <p className="clink">Email: <p className="con-details">cerebro@iiitvadodara.ac.in</p></p>
```
Comment (pAditya198) [inline src/Component/Footer/Footer.jsx:76]: don't use p inside p

File: src/Component/Footer/footcss.css [hunk 4]
```diff
@@ -15,19 +15,21 @@ footer {
   display: flex;
   justify-content: center;
   align-items: center;
-  padding-bottom: 3vh;
+  height: 117px;
+  padding-bottom: 30px;
 }
 
 .inslog {
+  /* padding-left: 10px; */
```
Comment (pAditya198) [inline src/Component/Footer/footcss.css:23]: remove unnecessary comments

Comment (shivam0110) [reply src/Component/Footer/footcss.css:23]: done

Comment (pAditya198) [reply src/Component/Footer/footcss.css:23]: you have to push the code too

Also remove all unnecessary logs and comments from both files

**Review by pAditya198 — APPROVED** [review]

**Comment by pAditya198** [comment]
LGTM. Merging

**Pull request merged**